readme/contribution/MYSQL_SETUP.md

```diff
@@ -0,0 +1,35 @@
+# Setup local MySQL with Docker
```

Comment:
There are too many files in root and these two setup don't seem important enough to be in the root. How above move them to docs/?

Reply:
+1 for this. I think we already have a `docs` folder at root and we can reuse it instead of creating an extra `readme` folder?

Reply:
Sounds good. Moved to the docs folder